README.md

```diff
@@ -1,28 +1,33 @@
 # EST Server
+
 The project is an open source implementation of the Enrollment over Secure Transport
-(EST) protocol as defined in [RFC 7030](https://www.rfc-editor.org/rfc/rfc7030.html).
+(EST) protocol as defined in [RFC 7030](https://www.rfc-editor.org/rfc/rfc7030.html)
+with added notes from [RFC 8951](https://www.rfc-editor.org/rfc/rfc8951) and
+[RFC 8996](https://www.rfc-editor.org/rfc/rfc8996).
```

Comment:
Are you sure we are compliant with those added notes?

Reply:
We definitely are with 8996, as it states it deprecates TLS 1.0 and TLS 1.1 on the server side. We start the server on TLS1.2. You already send everything back in base64 encoded bytes without setting the header to Content-Transfer-Encoding base64. Which is basically all of 8951.